### DISCUSSION TITLE
[EMERGENCY] Need help with my code (JS)

### DISCUSSION THREAD

**Discussion by Ray-hirano** [community/community#144398] [Programming Help]
### Select Topic Area

Question

### Body

https://github.com/Ray-hirano/silo-management-app
On the above repository, i am working on a minimal viable product for a silo management "software".

I am trying to get sets of buttons in the modal that could show the historical data (included in data/silo-data.json). However, it seems like it is not showing up.
Also, if possible, I want the first modal to show the trend depending on the couple most recent data (e.g. tempreture rising) but this part, I just don;t know how to implement. Thank you in advance for the help.

**Comment by mojerf** [comment]
@Ray-hirano its a private repository i guess
we can not access it

**Reply by Ray-hirano** [reply]
I vhanged it to public, can you see it now?
